src/index.html

```diff
@@ -5,8 +5,428 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="font">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="nav_container">
+        <div class="logo-img-container">
+          <img
+            class="logo_img"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active"href="http://localhost:8080/">apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="http://localhost:8080/">samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="http://localhost:8080/">smartphones</a>
+            </li>
+
+            <li class="nav__item" data-qa="nav-hover">
+              <a class="nav__link" href="http://localhost:8080/">laptops & computers</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="http://localhost:8080/">gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="http://localhost:8080/">tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="http://localhost:8080/">photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="http://localhost:8080/">video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main-page">
+      <div class="cards">
+        <div class="card" data-qa="card" >
+          <img
+          src="./images/imac.jpeg"
+          alt="laptop"
+          class="card__image"
+          >
```

Comment:
please add spacing to attributes
```suggestion
          <img
            src="./images/imac.jpeg"
            alt="laptop"
            class="card__image"
          >
```